plugin/core/registry.py

```diff
@@ -21,14 +21,14 @@
 windows = WindowRegistry()
 
 
-def best_session(view: sublime.View, sessions: Iterable[Session], point: int | None = None) -> Session | None:
+def best_session(view: sublime.View, sessions: list[Session], point: int | None = None) -> Session | None:
```

Comment:
I think this change for the parameter type is unnecessary here because `list` is still an `Iterable`, so this just makes the function more restrictive where it doesn't need to be. And `Iterable` would be more inline with the usage in `max`, which takes an iterable https://docs.python.org/3/library/functions.html#max

But frankly it doesn't really matter and I think nobody payed much intention of this throughout the rest of the code base.